examples_32bit_test.go

```diff
@@ -0,0 +1,39 @@
+//go:build 386
+
+package safecast_test
+
+// The tests in examples_test.go are the ones that are not architecture dependent
+// The tests in examples_32bit_test.go are for 32-bit systems
+// The tests in examples_64bit_test.go are for 64-bit systems
+//
+// The architecture dependent files cover the difference when dealing with [safecast.ToInt]
+// The error message is different on 32-bit and 64-bit systems
+// The max is 9223372036854775807 on 64-bit and 2147483647 on 32-bit
+//
+// The examples could have been skipped for 32-bit systems,
+// but I wanted the Examples to be launched on this architecture.
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/ccoveille/go-safecast"
+)
+
+func ExampleToInt() {
+	a := uint64(42)
+	i, err := safecast.ToInt(a)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
This is me when I see an error written to STDOUT instead of STDERR 😄 :

![angry](https://makeameme.org/media/templates/250/angry.jpg).

can you `fmt.Fprintln(os.Stderr, err)` please !

Reply:
Same for all examples

Reply:
should be log.Printf() really

Reply:
I totally agree with you, however, I would consider using `panic(err)` in examples rather than `fmt.Println` for errors.

Reply:
Thabks guys, I will check and change

Reply:
I checked and none works

- panic would lead to a panic, so the automatic test of the example file will fail
- log.Printn would dump a log date in the ouput

Please note I'm validating the examples with `// Output: ` syntax

https://pkg.go.dev/testing#hdr-Examples

> The package also runs and verifies example code. Example functions may include a concluding line comment that begins with "Output:" and is compared with the standard output of the function when the tests are run. (

I looked for examples in the stdlib

https://pkg.go.dev/strconv@go1.23.3#example-Unquote

I won't play with this and a non-default logger
https://pkg.go.dev/log#Lmsgprefix

I will keep my approach of fmt.Println

Reply:
@ldemailly @yassinebenaid did you see my explanation ?